sensors/tests/test_environment_sensor.cpp

```diff
@@ -111,7 +111,7 @@ SCENARIO("read temperature and humidity values") {
                             can_msg.message);
                     float check_data =
                         fixed_point_to_float(response_msg.sensor_data, 15);
-                    float expected = 44.20312;
+                    float expected = 88.40625;
```

Comment:
How did these values change?

Reply:
Well, we were not correctly converting fixed points to floats - we were halving the value. This data now matches what it should be.

Reply:
Binary strikes again 😫